src/resources/Notifications/Subscriptions/SubscriptionsInterfaces.ts

```diff
@@ -0,0 +1,53 @@
+import {SubscriptionFrequencyEnum, SubscriptionStatusHealthIndicatorEnum, SubscriptionTypeEnum} from '../../Enums';
+
+export interface EditSubscriptionModel {
+    description: string;
+    enabled: boolean;
+    frequency: SubscriptionFrequencyEnum;
+    name: string;
+    parameters: SubscriptionParameters;
+    pattern: SubscriptionPatternModel;
+    type: SubscriptionTypeEnum;
+}
+
+export interface SubscriptionModel {
```

Comment:
The properties in `EditSubscriptionModel` seem to be included by `SubscriptionModel`. Maybe we could try to make `EditSubscriptionModel` generic and then `export interface SubscriptionModel extends ***`

Reply:
Will Do